# Define the goal
float32 desired_yaw
---
# Define the result
float32 achieved_yaw
---
# Define a feedback message
float32 percent_complete
